src/common/i18n/translations/zh.ts

```diff
@@ -25,6 +25,9 @@ export const zh: Translations = {
     or: "或",
     enterIdNumber: "输入证件号码",
     check: "查看",
+    goToStatistics: "前往统计",
+    nationalityInputLabel: "国籍 (非新加坡人)",
```

Comment:
I'll suggest the following instead, so we'll say that you're a traveller, and hence not a singaporean HAHa
```suggestion
    nationalityInputLabel: "国籍 (游客)",
```

Reply:
According to Eida, she has some concern since not everyone that comes to Singapore === tourist. e.g, visitors who come to see a doctor. So cannot say tourist. wdyt?

Reply:
I see I see! Ok, I think let's leave it first hahaha